lstchain/scripts/lstchain_dl1ab_pedestal_cleaning.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
```

Comment:
Why a different script? It should rather be an option of the existing dl1ab script, right?

Reply:
This solution will be better. So after PR #543 I can add this option, right?

Reply:
> This solution will be better. So after PR #543 I can add this option, right?

ah, ok, right, better wait for #543 which should be nearly ready I think.

Reply:
now that #543 is merged, can we implement this as an option of the  `lstchain_dl1ab.py` script?